src/Sulu/Bundle/AdminBundle/Resources/js/views/List/List.js

```diff
@@ -102,18 +110,32 @@ class List extends React.Component<ViewProps> {
     buildListStoreOptions(
         apiOptions: Object,
         attributes: Object,
-        routerAttributesToListStore: {[string | number]: string}
+        routerAttributesToListStore: {[string | number]: string},
+        resourceStorePropertiesToListStore: {[string | number]: string},
+        resourceStore: ?ResourceStore
     ) {
         const listStoreOptions = apiOptions ? apiOptions : {};
 
         routerAttributesToListStore = toJS(routerAttributesToListStore);
         Object.keys(routerAttributesToListStore).forEach((key) => {
-            const attributeName = routerAttributesToListStore[key];
-            const listOptionKey = isNaN(key) ? key : routerAttributesToListStore[key];
+            const listOptionKey = routerAttributesToListStore[key];
+            const attributeName = isNaN(key) ? key : routerAttributesToListStore[key];
```

Comment:
I think this change is worth being mentioned in the `UPGRADE.md` file. Also, I would say that if you do this change for the `List` view, it should also be done for the `Form` view, otherwise we are inconsistent, which is really confusing.

Reply:
done and rebased